packages/editor/src/components/editor-interface/index.js

```diff
@@ -192,7 +190,9 @@ export default function EditorInterface( {
 				isLargeViewport &&
 				showBlockBreadcrumbs &&
 				mode === 'visual' && (
-					<BlockBreadcrumb rootLabelText={ documentLabel } />
+					<BlockBreadcrumb
+						rootLabelText={ decodeEntities( postTypeLabel ) }
```

Comment:
Wouldn't it make sense to decode this label in the `BlockBreadcrumb` component and before the actual value is rendered?

My thinking here is similar to escaping data: you want to escape when you echo, not before. Ref: https://developer.wordpress.org/apis/security/escaping/#escaping-functions